$NetBSD: patch-ax,v 1.1 2007/03/17 14:33:13 rillig Exp $

gcc4 on NetBSD 4 complains:
c_shell.c:52: warning: missing sentinel in function call
c_shell.c:54: warning: missing sentinel in function call

--- src/c_shell.c.orig	2000-03-15 23:13:28.000000000 +0100
+++ src/c_shell.c	2007-03-17 15:31:31.000000000 +0100
@@ -49,9 +49,9 @@ c_shell (TARGLIST)
 
 	if ((pid = fork()) == 0) {
 		if (shellout)
-			execlp(shell, shell, 0);
+			execlp(shell, shell, (char *)0);
 		else
-			execlp(shell, shell, "-c", mbuf, 0);
+			execlp(shell, shell, "-c", mbuf, (char *)0);
 
 		/* exec failed if we make it here */
 		sprintf(mbuf, "c_shell: can't run shell \"%s\"", shell);
